packages/node/src/index.ts

```diff
@@ -85,14 +85,29 @@ export { getModuleFromFilename } from './module';
 export { enableAnrDetection } from './integrations/anr/legacy';
 
 import { Integrations as CoreIntegrations } from '@sentry/core';
+import type { Integration } from '@sentry/types';
 
 import * as Handlers from './handlers';
 import * as NodeIntegrations from './integrations';
 import * as TracingIntegrations from './tracing/integrations';
 
 const INTEGRATIONS = {
   ...CoreIntegrations,
-  ...NodeIntegrations,
+  ...(NodeIntegrations as {
+    Console: Integration;
```

Comment:
TS complains about this otherwise, somehow 😬 with `TS4023: Exported Variable <x> has or is using name <y> from external module but cannot be named`. No idea, but I think this is fine for now...